specs/commandline/blob.md

```diff
@@ -26,20 +26,21 @@ Flags:
 ### notation blob sign
 
 ```text
-Produce a detached signature for a given blob
+Produce a detached signature for a given blob. Signature file will be written to the currently working directory with blob file name + ".sig" + signature format as the file extension. For example, signature file name for "myBlob.bin" will be "myBlob.bin.sig.jws" for JWS signature format or "myBlob.bin.sig.cose" for COSE signature format.
 
 Usage:
-  notation blob sign [flags] -n my-blob-signature <blob_path>
+  notation blob sign [flags] <blob_path>
 
 Flags:
-  -n,  --signature-name string      friendly name for the detached signature. Signature file will be written to the currently working directory with this name plus ".sig" plus signature format as the file extension
+  -sd, --signature-directory string optional path where the blob signature needs to be placed (default: currently working directory) 
```

Comment:
The short-hand `-sd` is not valid. It will be parsed as `-s` and `-d`.

Reply:
Will fix it. Thanks